src/Compilers/CSharp/Test/Emit/CodeGen/CodeGenAwaitForeachTests.cs

```diff
@@ -9929,11 +9929,7 @@ public static async Task Main()
     }
 }";
             CreateCompilationWithTasksExtensions(new[] { source, AsyncStreamsTypes }, parseOptions: TestOptions.Regular9)
-                .VerifyDiagnostics(
-                    // (9,33): error CS0186: Use of null is not valid in this context
-                    //         await foreach (var i in (IAsyncEnumerable<int>)null)
-                    Diagnostic(ErrorCode.ERR_NullNotValid, "(IAsyncEnumerable<int>)null").WithLocation(9, 33)
-                    );
+                .VerifyEmitDiagnostics();
```

Comment:
It would be even better to verify execution (throws NRE). Comment also applies to other tests that no longer have errors

Reply:
will consider in followup.